sw/device/silicon_creator/rom/rom.c

```diff
@@ -653,6 +653,40 @@ static rom_error_t rom_boot(const manifest_t *manifest, uint32_t flash_exec) {
   // In a normal build, this function inlines to nothing.
   stack_utilization_print();
 
+  // (Potentially) Execute the immutable ROM_EXT partition.
+  if (otp_read32(OTP_CTRL_PARAM_CREATOR_SW_CFG_IMMUTABLE_ROM_EXT_EN_OFFSET) ==
```

Comment:
You probably need a hardened check on the disabled case:

```
uint32_t enabled = otp_read32(OTP_CTRL_PARAM_CREATOR_SW_CFG_IMMUTABLE_ROM_EXT_EN_OFFSET);
if (launder32(enabled) == kHardenedBoolTrue) {
   ... check hash & execute ...
} else {
  HARDENED_CHECK_NE(enabled, kHardenedBoolTrue);
}
```

Reply:
Fixed.